.jenkins/build.sh

```diff
@@ -27,10 +27,12 @@ unzip img_align_celeba.zip -d /home/ubuntu/facebook/datasets/celeba > null
 mkdir data/
 curl https://s3.amazonaws.com/pytorch-tutorial-assets/iris.data --output data/iris.data
 
-# Download dataset for beginner_source/chatbot_tutorial.py
+# Download dataset and model for beginner_source/chatbot_tutorial.py
 curl https://s3.amazonaws.com/pytorch-tutorial-assets/cornell_movie_dialogs_corpus.zip --output cornell_movie_dialogs_corpus.zip
 mkdir -p beginner_source/data
 unzip cornell_movie_dialogs_corpus.zip -d beginner_source/data/ > null
+mkdir -p "beginner_source/data/save/model7/cornell movie-dialogs corpus/2-2_500"
+curl https://s3.amazonaws.com/pytorch-tutorial-assets/4000_checkpoint.tar --output "beginner_source/data/save/model7/cornell movie-dialogs corpus/2-2_500/4000_checkpoint.tar"
```

Comment:
fyi, this is available at https://download.pytorch.org/models/tutorials/4000_checkpoint.tar